pages/common/topydo.md

```diff
@@ -0,0 +1,28 @@
+# topydo
+
+> A powerful todo list application using the todo.txt format.
+> More information: <https://github.com/topydo/topydo>.
+
+- Add todo to the project "tldr" and the context "github":
+
+`topydo add "merge pull request +tldr @github"`
+
+- Add todo which should be done till tomorow with a "A" priority:
+
+`topydo add "(A) call mom due:1d"`
+
+- Add todo which should be done till friday:
+
+`topydo add " due:fri"`
+
+- Add non-strict repeating todo (next due = now + rec):
+
+`topydo add "water flowers due:mon rec:1w"`
+
+- Add strict repeating todo (next due = currentdue + rec):
+
+`topydo add "pay rent due:2020-01-01 rec:+1m"`
```

Comment:
Perhaps this will read better?
```suggestion
- Add a strict repeating todo (next due = currentdue + rec):

`topydo add "{{todo_message}} due:{{2020-01-01}} rec:{{+1m}}"`
```